tests/Controller/ProductApiTest.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Tests\Controller;
+
+use Lakion\ApiTestCase\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class ProductApiTest extends JsonApiTestCase
```

Comment:
We could make this tests final. I know, that not all of them are, but we should make them all final.